tests/sxg_buffer_test.cc

```diff
@@ -14,20 +14,16 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include "libsxg/sxg_buffer.h"
-
 #include <string>
 
 #include "gtest/gtest.h"
 #include "libsxg/internal/sxg_buffer.h"
+#include "libsxg/sxg_buffer.h"
+#include "test_util.h"
 
 namespace {
 
-std::string BufferToString(const sxg_buffer_t& buf) {
-  // Casting from uint8_t* to char* is legal because we confirmed char to have 8
-  // bits.
-  return std::string(reinterpret_cast<const char*>(buf.data), buf.size);
-}
+using sxg_test::BufferToString;
```

Comment:
Fully qualify names (i.e. add `::` prefix) in `using` declarations.
https://abseil.io/tips/119, https://abseil.io/tips/130

Reply:
Done.